### ISSUE TITLE
Sorting of features

### ISSUE THREAD

**Issue by bguil** [kirianguiller/conllup-py#1]
According to https://universaldependencies.org/format.html:
> _In sorting, uppercase letters are considered identical to their lowercase counterparts._

For instance, [here](https://github.com/UniversalDependencies/UD_French-Sequoia/blob/0bf24efd2adc42dbdbae14423ec77e2e70b6db0e/fr_sequoia-ud-dev.conllu#L637):
```
16	premières	premier	ADJ	_	Gender=Fem|Number=Plur|NumType=Ord	15	amod	_	_
```
`Number` appears before `NumType` because `number` < `numtype` even if `NumType` < `Number`

In `writeConlluFile`, the output is in the other way.

**Comment by kirianguiller** [comment]
fixed in #2 , thank you @bguil  !

**Comment by kirianguiller** [comment]
added some testing for recently modified function in [8eae4c8](https://github.com/kirianguiller/conllup-py/commit/8eae4c83e30787a812aa93806b58aaf3a8efdb14)
